sceptre/template_handlers/http.py

```diff
@@ -66,18 +77,57 @@ def handle(self):
 
         return template
 
-    def _get_template(self, url):
+    def _get_template(self, url, retries, timeout):
         """
         Get template from the web
         :param url: The url to the template
         :type: str
-        :returns: The body of the CloudFormation template.
-        :rtype: str
+        :returns: The number of retry attempts.
+        :rtype: int
+        :returns: The timeout for the session in seconds.
+        :rtype: int
```

Comment:
You know, with type annotations, you wouldn't need even half of these...

Reply:
those are supposed to be param not returns.  i have fixed.